tests/tests_suite_test.go

```diff
@@ -128,6 +128,9 @@ func (s *newSspStrategy) Init() {
 			TektonPipelines: &sspv1beta1.TektonPipelines{
 				Namespace: s.GetNamespace(),
 			},
+			TektonTasks: &sspv1beta1.TektonTasks{
+				Namespace: s.GetNamespace(),
```

Comment:
Can you use a different namespace here, so we can test that deployment to other namespaces works.

Reply:
We'll set a different namespace only to `TektonTasks`? or `TektonPipelines` as well?

To set a different namespace here, will we create `func (s *newSspStrategy) GetTektonTasksNamespace() string` and `func (s *newSspStrategy) GetTektonPipelinesNamespace() string`) if needed?

Reply:
@ksimon1 do the tasks and pipelines have to be in the same namespace?

Reply:
they don't have to be, but if they are, user has to create clusterResolver object

Reply:
Done.

Reply:
@ksimon1 , I don't understand. The `ClusterResolver` needs to be created only if pipelines and tasks are in the same namespace? So if they are in different namespaces, the `ClusterResolver` is not needed?

I think I leave it to you to say in what namespace the `TektonTasks` should be deployed for functional tests.